### ISSUE TITLE
Setting up Prisma

### ISSUE THREAD

**Issue by lamara11** [amosproj/amos2024ss01-xcelerator-demo-app#43]
## User story

1. As a developer
2. I want to set up Prisma in our project
3. So that I can efficiently manage database operations

## Acceptance criteria

- [x] Prisma is successfully installed and configured in the project.
- [x] Prisma schema is defined to model database tables according to project requirements.

## Definition of done (DoD)

- [ ] Added only after week 5
- [ ] The same for all features
- [ ] Here goes the project specific part

## DoD general criteria

- [ ] Feature has been fully implemented
- [ ] Feature has been merged into the mainline
- [ ] All acceptance criteria were met
- [ ] Product owner approved features
- [ ] All tests are passing
- [ ] Developers agreed to release
